tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -8,9 +8,72 @@
 #include <string>
 #include <string_view>
 
+using namespace std;
+
 // TODO: fill in tests
-template std::back_insert_iterator<std::string> std::vformat_to(std::back_insert_iterator<std::string>,
-    const std::locale&, std::string_view, std::format_args_t<std::back_insert_iterator<std::string>, char>);
+template back_insert_iterator<string> std::vformat_to(
+    back_insert_iterator<string>, const locale&, string_view, format_args_t<back_insert_iterator<string>, char>);
+
+
+int main() {
+    string output_string = "";
+
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "f", make_format_args());
+    assert(output_string == "f");
+
+    output_string.clear();
+    vformat_to(back_insert_iterator(output_string), locale::classic(), "format", make_format_args());
+    assert(output_string == "format");
+
+    // test escaped opening curlies

```

Comment:
Mega-nitpick: we call these "open curlies" in the product code instead of "opening".

Reply:
what we call them opening curls!

Reply:
Wow, I'm completely wrong - what was I looking at??!?